testing/README.md

```diff
@@ -0,0 +1,45 @@
+# Running tests locally
+
+## Prerequisites
+
+Some additional steps are required for running tests.
+
+Firstly, install optional dependencies (if still not):
+
+```shell
+pip install mlcube_docker openvino==2023.0.1
+```
+
+Second, download and unpack the following testing data to `testing/` subfolder:
+
+```shell
+# it's assumed you are in `GaNDLF/` repo root directory
+cd testing/
+wget https://upenn.box.com/shared/static/y8162xkq1zz5555ye3pwadry2m2e39bs.zip
+unzip $FILENAME
+```
```

Comment:
Got it, thanks!
True, now I found how this is executed (and why it was not executed in my case).
As the data downloading & csv building are organized as individual tests, pytest run them only when they are included in the scope; say, when we run all tests by `pytest .`
However, when I run only subset (say, `pytest -k "classification"` or any specific test directly), data is not downloaded.
So, when I tried to run some tests not having data downloaded, I faced troubles:)

I've added a fix: now data download & preparation functions are called by `pytest_sessionstart` hook in `conftest.py`. Thus, every time `pytest` session is started, it would try to download data if necessary.